models/stg_mailchimp.yml

```diff
@@ -0,0 +1,586 @@
+version: 2
+
+models:
+
+  - name: stg_mailchimp__automation_activities
+    description: Each record represents an activity taken in relation to a automation email.
+    columns:
+      - name: activity_id
+        description: The ID of the activity/event.
+        tests: 
+          - not_null
+          - unique
+
+      - name: action_type
+        description: >
+          One of the following actions: 'open', 'click', or 'bounce'
+
+      - name: automation_email_id
+        description: The ID for the automation email that relates to the activity.
+
+      - name: member_id
+        description: The ID of the member the activity relates to.
+
+      - name: list_id
+        description: The ID of the list the email was sent to.
+
+      - name: activity_timestamp
+        description: The timestamp when the activity occurred.
+
+      - name: ip_address
+        description: The IP address recorded for the action.
+
+      - name: url
+        description: If the action is a 'click', the URL on which the member clicked.
+
+      - name: bounce_type
+        description: >
+          If the action is a 'bounce', the type of bounce received: 'hard', 'soft'.
+
+  - name: stg_mailchimp_automation_emails
```

Comment:
It looks like for the majority of the tables in the `stg_mailchimp.yml` the naming has not been updated. You can see this causes `WARNING` messages when running `dbt run` or `dbt test` as the tests are not technically associated with a model. This should resolve itself once you update the naming convention of the stg models in the yml to point to the double underscore names.
![image](https://user-images.githubusercontent.com/74217849/136129144-2147a971-1c33-4359-a532-0b3c549726c4.png)
You can also see above ^ that when running tests it will only run for the properly named models.